src/modules/dashboard/components/ColonyEvents/ColonyEventsListItem.tsx

```diff
@@ -77,6 +79,13 @@ const ColonyEventsListItem = ({
     ({ address }) => address === nativeTokenAddress,
   );
 
+  const getFormattedRole = () => {
+    const roleNameMessage = { id: `role.${role}` };
+    return `${setTo ? 'assigned' : 'removed'} the ${formatMessage(
+      roleNameMessage,
+    ).toLowerCase()} permission`;
+  };
+
```

Comment:
This is the same as the one in the `ActionsListItem`.

Since it's duplicated code, please refactor this out and move it into utils/events so that it can be shared